float32[10] linear_vel
float32[10] angular_vel
